### PR TITLE
Cp analysis

### PR DESCRIPTION
**Pull request by carla-terboven**
- [x] TODOs for app view: show sampleID in columns section and set the widget layouts for xl and xxl size
- [ ] reprocess automatically (see mattermost chat)

### CODE REVIEW

File: src/nomad_chemical_energy/apps/__init__.py [hunk 1]
```diff
@@ -855,3 +855,373 @@
         }
```
Comment (RoteKekse) [inline src/nomad_chemical_energy/apps/__init__.py:855]: I think it makes sense to add a widget for authors and datetime maybe?

File: src/nomad_chemical_energy/parsers/ce_nome_parser.py [hunk 2]
```diff
@@ -151,6 +152,12 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
         environment_ref = find_sample_by_id(archive, environment_id)
         setup_ref = find_sample_by_id(archive, setup_id)
 
+        if "TITLE" in metadata:
+            label = metadata.get("TITLE")
+            if "OER CP" in label:
```
[the remark this answers is not in the document]
Comment (carla-terboven) [reply src/nomad_chemical_energy/parsers/ce_nome_parser.py:157]: I don't think we can use time here (see mattermost chat).

File: src/nomad_chemical_energy/apps/__init__.py (hunk 1 above)
Comment (carla-terboven) [reply src/nomad_chemical_energy/apps/__init__.py:855]: :white_check_mark:

**Comment by RoteKekse** [comment]
Looks good, this means only one cp analysis per upload. in some sense that also makes sense. if you upload files from another day you still wanna know how the latest stability is i guess :).

**Pull request merged**